spec/controllers/articles_controller_spec.rb

```diff
@@ -1,22 +1,22 @@
 require 'spec_helper'
 
 RSpec.describe ArticlesController, type: :controller do
-  describe '#index' do
+  describe 'GET #index' do
     describe 'show the list to articles saved' do
     end
   end
 
-  describe '#show' do
+  describe 'GET #show' do
     it 'show the articles' do
     end
   end
 
-  describe '#new'do
+  describe 'GET #new'do
     it 'create new post' do
     end
   end
 
-    describe ' #create' do
+    describe 'GET #create' do
```

Comment:
`describe 'POST #create' do`

Methods `POST` persist information to the database, here the object is manipulated to get the expected result